### PR TITLE
Make improvements to the UNO game

### PR DESCRIPTION
**Pull request by KrisXV**
This includes:
- adding timestamps to all playerlogged actions
- changing the background color from green to blue
- adding proper CSS to the buttons
- adding a "Join as a Spectator" button to the signups
- fixing grammatical errors and redundancies
- making the height of the "UNO!" button taller so players can click it more easily when chat scrolls
- increasing the default playercap from 6 to 12
- adding an `/uno setcap` command so people can adjust the playercap without having to end the game and create a new one (or use eval like Hoeen)

### CODE REVIEW

File: server/chat-plugins/uno.ts [hunk 1]
```diff
@@ -140,28 +140,26 @@ export class UnoGame extends Rooms.RoomGame {
 		if (this.state === 'signups') {
 			connection.sendTo(
 				this.room,
-				`|uhtml|uno-${this.gameNumber}|<div class="broadcast-green">` +
+				`|uhtml|uno-${this.gameNumber}|<div class="broadcast-blue">` +
 				`<p style="font-size: 14pt; text-align: center">A new game of <strong>UNO</strong> is starting!</p>` +
-				`<p style="font-size: 9pt; text-align: center"><button name="send" value="/uno join">Join</button>` +
-				`<br />Or use <strong>/uno join</strong> to join the game.</p>` +
+				`<p style="font-size: 9pt; text-align: center"><button class="button" name="send" value="/uno join">Join and play</button> ` +
```
Comment (Zarel) [inline server/chat-plugins/uno.ts:145]:
```suggestion
				`<p style="font-size: 9pt; text-align: center"><button class="button" name="send" value="/uno join"><strong>Join and play</strong></button> ` +
```

File: server/chat-plugins/uno.ts [hunk 2]
```diff
@@ -140,28 +140,26 @@ export class UnoGame extends Rooms.RoomGame {
...
+				`<button class="button" name="send" value="/uno spectate">Watch</button></p>` +
 				`${this.suppressMessages ?
-					`<p style="font-size: 6pt; text-align: center">Game messages will be shown to only players. ` +
-					`If you would like to spectate the game, use <strong>/uno spectate</strong></p>` : ''}</div>`
+					`<p style="font-size: 6pt; text-align: center">Game messages will only be shown to players.` : ''}</div>`
```
Comment (Zarel) [inline server/chat-plugins/uno.ts:148]:
```suggestion
					`<p style="font-size: 6pt; text-align: center">Game messages won't show up unless you're playing or watching.` : ''}</div>`
```

**Comment by KrisXV** [comment]
did @Zarel

**Comment by Zarel** [comment]
Cool! I like the new design a lot more.

**Pull request merged**